src/components/Summary/GenericSummary.js

```diff
@@ -0,0 +1,103 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Card from 'components/Card';
+import ErrorMessage from 'components/ErrorMessage';
+import FormStepSummary from 'components/FormStepSummary';
+import {LayoutColumn} from 'components/Layout';
+import Loader from 'components/Loader';
+import LogoutButton from 'components/LogoutButton';
+import Price from 'components/Price';
+import SummaryConfirmation from 'components/SummaryConfirmation';
+import {SUBMISSION_ALLOWED} from 'components/constants';
+
+const GenericSummary = ({
+  title,
+  submissionAllowed,
+  summaryData,
```

Comment:
```suggestion
  summaryData = [],
```
It's the proptype is not `.isRequired`, but we do call `.map` on it.